server/core/config/valid/repo_cfg.go

```diff
@@ -91,6 +92,20 @@ func isRegexAllowed(name string, allowedRegexpPrefixes []string) bool {
 	return false
 }
 
+func (r RepoCfg) AutoDiscoverEnabled(defaultAutoDiscoverMode AutoDiscoverMode) bool {
```

Comment:
How about some function description comments for this?

Reply:
Done!